wsproxy/websocket_proxy.go

```diff
@@ -6,15 +6,75 @@ import (
 	"net/http"
 	"strings"
 
-	log "github.com/Sirupsen/logrus"
+	"github.com/Sirupsen/logrus"
 	"github.com/gorilla/websocket"
 	"golang.org/x/net/context"
 )
 
-var (
-	MethodOverrideParam = "method"
-	TokenCookieName     = "token"
-)
+// MethodOverrideParam defines the special URL parameter that is translated into the subsequent proxied streaming http request's method.
+var MethodOverrideParam = "method"
```

Comment:
Is there a reason `TokenCookieName` gets a deprecation warning but this does not?